erpnext/accounts/doctype/account/account_tree.js

```
@@ -1,85 +1,280 @@
+			disable_onchange: true,
+		},
+	],
+	root_label: "Accounts",
+	get_tree_nodes: "erpnext.accounts.utils.get_children",
+	on_get_node: function (nodes, deep = false) {
+		if (frappe.boot.user.can_read.indexOf("GL Entry") == -1) return;
+
+		let accounts = [];
+		if (deep) {
+			// in case of `get_all_nodes`
+			accounts = nodes.reduce((acc, node) => [...acc, ...node.data], []);
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)